src/__tests__/__snapshots__/index.js.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`preval a-bunch-of-checks: a-bunch-of-checks 1`] = `
+exports[`unknown plugin a-bunch-of-checks: a-bunch-of-checks 1`] = `
```

Comment:
I don't know why all these names have changed in the snapshots.  Probably related to updated versions of things, but not sure which one might have caused it.  [`babel-plugin-codegen` does not appear to be affected like this though?](https://github.com/kentcdodds/babel-plugin-codegen/blob/9bb9dfd26a6ba55654bd493ee73b9778f8e66523/src/__tests__/__snapshots__/index.ts.snap#L3)

Reply:
I'm still investigating this, but I have a suspicion that the correct name may come back once the conversion is complete (or rather when `index.js` is done).  I think we should ignore this for now and keep moving, then circle back to it later.

Reply:
I have [raised a seperate issue for this](https://github.com/OpenSourceRaidGuild/babel-plugin-preval/issues/5).